src/sentry/api/serializers/models/project.py

```diff
@@ -475,6 +480,55 @@ def serialize(self, obj, attrs, user):
         return data
 
 
+##########
```

Comment:
What are these comments for?

Reply:
I think you're looking at an older version? I don't see this line anymore